data-table/__tests__/table.js

```diff
@@ -551,3 +551,35 @@ describe('Test cell getters and setters', () => {
         expect(numErrorsVisible()).toEqual(0);
     });
 });
+
+describe("test input types", () => {
+    test("type is boolean", () => {
```

Comment:
test looks good! In general, it's better to group tests with similar tests. Since there's only one `test` in this `describe`, is there another `describe` where it might fit better?